staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode_test.go

```diff
@@ -37,38 +39,47 @@ func TestDecode(t *testing.T) {
 		return b
 	}
 
-	for _, tc := range []struct {
+	type UncheckedValue struct{}
```

Comment:
It's worth explaining the purpose and usage of this type in a comment.

Reply:
Added a comment.